src/app/_components/CTAButtonGroup.tsx

```diff
@@ -0,0 +1,37 @@
+import { clsx } from 'clsx'
+
+import { Button } from '@/components/Button'
+
+import type { CTAProps } from '@/types/sharedProps/ctaType'
+
+export type CTAButtonGroupProps = {
+  cta: CTAProps | [CTAProps, CTAProps]
+}
+
+export function CTAButtonGroup({ cta }: CTAButtonGroupProps) {
+  const ctaArray = Array.isArray(cta) ? cta : [cta]
+
+  const gridClasses = clsx(
+    'grid gap-4',
+    ctaArray.length === 1
+      ? 'lg:grid-cols-1'
+      : 'sm:grid-cols-2 sm:gap-3 lg:grid-cols-1 lg:gap-4',
+  )
+
+  return (
+    <div className={gridClasses}>
+      {ctaArray.map(
+        (button, index) =>
+          button && (
+            <Button
+              key={index}
```

Comment:
https://react.dev/learn/rendering-lists#why-does-react-need-keys - they recommend not using index as a key

```suggestion
              key={button.href}
```